src/Flex.php

```diff
@@ -250,6 +253,21 @@ public function executeAutoScripts(Event $event)
         $this->io->write($this->postInstallOutput);
     }
 
+    public function populateCacheDir(InstallerEvent $event)
+    {
+        if (extension_loaded('curl') && class_exists(Prestissimo::class, false)) {
```

Comment:
Are you sure `Prestissimo` is the right class? I can't find it on https://github.com/hirak/prestissimo

Reply:
missing alias, fixed thanks